project/Dependencies.scala

```diff
@@ -369,6 +371,9 @@ object Dependencies {
       "javax.activation"           % "javax.activation-api"   % "1.2.0",
       "jakarta.activation"         % "jakarta.activation-api" % "1.2.1",
       "com.thoughtworks.paranamer" % "paranamer"              % "2.8",
+      mockitoCore,
+      "org.objenesis" % "objenesis"        % "2.6",
+      "net.bytebuddy" % "byte-buddy-agent" % "1.10.5"
```

Comment:
Maybe we should extract to a new `Version.ByteBuddy = "1.10.5"` (see change in line 93 above). Or even add `   private val byteBuddyAgent              = "net.bytebuddy" % "byte-buddy-agent" % ...` above so both deps are together.

Reply:
They are not used in other places, they are being pulled transitively by mockitoCore. 

Anyway, I will improve it, but on another PR. I don't want to delay the release because of that as a new commit will require another build.